packages/langium/src/service-registry.ts

```diff
@@ -4,38 +4,41 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
-import type { LangiumServices } from './services.js';
+import type { LangiumCoreServices } from './services.js';
 import { UriUtils, type URI } from './utils/uri-util.js';
 
 /**
- * The service registry provides access to the language-specific services. These are resolved
+ * The core service registry provides access to the core language-specific services. These are resolved
  * via the URI of a text document.
  */
-export interface ServiceRegistry {
+export interface CoreServiceRegistryI {
```

Comment:
I've added it since below we have a conflict with `CoreServiceRegistry` which implements this interface, but I'm not particularly fond of the name choices so far (I made a comment about this earlier as well). I wouldn't be opposed to moving the initial around, but I'm more curious if the choice of making a 'Core' service makes sense.

Reply:
Flipped the I around in the last changes.